code/modules/mob/living/silicon/robot/inventory.dm

```diff
@@ -3,7 +3,7 @@
 
 //Returns the thing in our active hand (whatever is in our active module-slot, in this case)
 /mob/living/silicon/robot/get_active_hand()
-	return module_active
+	return module_active ? module_active : current_grab

```

Comment:
Doesn't really matter, but this could be shortened to `return module_active || current_grab`

Reply:
I feel like this is a bit more readable, but `foo || bar` always confuses me a bit.